src/git-istage/Application.cs

```diff
@@ -152,7 +152,11 @@ private void UpdateRepository()
             var changes = _viewStage
                 ? _repository.Diff.Compare<TreeChanges>(tipTree, DiffTargets.Index)
                 : _repository.Diff.Compare<TreeChanges>(null, true);
-            var paths = changes.Select(c => c.Path).ToArray();
+
+            var paths = changes
+                .Where(p => (p.Mode != Mode.SymbolicLink && p.OldMode != Mode.SymbolicLink))
+                .Select(c => c.Path).ToArray();
```

Comment:
Is skipping them entirely the right behavior? Are changes on files with junctions modelled as two entries? Or should we just map path?

Reply:
In my case, I junctioned 5GB directory with 100's K of files between git worktrees. Completely unnecessary comparison which was causing an exception from LibGit2Sharp ` .. change is a directory ..` or something along those lines